certified-connectors/IA-Connect JML/apiDefinition.swagger.json

```diff
@@ -6685,167 +6882,1261 @@
+            "description": "Inputs for IA-Connect AzureADv2IsUserInAzureADUserGroup action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "ObjectId": {
+                  "type": "string",
+                  "title": "User object Id or UPN",
+                  "description": "The Id of an Azure Active Directory user. You can specify a user by UPN (e.g. user@mydomain.onmicrosoft.com) or ObjectId (e.g. 01234567-0123-0123-0123-0123456789ab)."
+                },
+                "GroupObjectId": {
+                  "type": "string",
+                  "title": "Group object Id or display name",
+                  "description": "The Id or display name of an Azure Active Directory group. You can specify a group by Display Name (e.g. \"Finance users\") or ObjectId (e.g. 01234567-0123-0123-0123-0123456789ab)."
+                },
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "ObjectId",
+                "GroupObjectId",
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "UserIsInGroup": {
+                  "type": "boolean",
+                  "title": "User is in group",
+                  "description": "Whether the user is a member of the Azure AD group."
+                }
+              }
+            }
+          }
+        },
+        "operationId": "AzureADv2IsUserInAzureADUserGroup"
+      }
+    },
+    "/PowerShellAzureADv2/AzureADv2AddUserToGroup": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Add Azure AD user to group",
+        "description": "Add an Azure Active Directory user (or other object) to an existing Azure Active Directory group.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "AzureADv2AddUserToGroup",
+            "description": "Inputs for IA-Connect AzureADv2AddUserToGroup action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "UserObjectId": {
+                  "type": "string",
+                  "title": "User object Id or UPN",
+                  "description": "The Id of an Azure Active Directory user. You can specify a user by UPN (e.g. user@mydomain.onmicrosoft.com) or ObjectId (e.g. 01234567-0123-0123-0123-0123456789ab)."

```

Comment:
@SimonBondyUltima secre-look-a-like-values are not allowed, there're a few of them in descriptions. Could you please replace them with some string value? E.g. `ObjectId (e.g. UUID/GUID value)`